osbs/build/spec.py

```diff
@@ -189,7 +189,6 @@ def __init__(self):
         self.koji_kerberos_principal = BuildParam("koji_kerberos_principal", allow_none=True)
         self.koji_kerberos_keytab = BuildParam("koji_kerberos_keytab", allow_none=True)
         self.flatpak = BuildParam("flatpak", default=False)
-        self.flatpak_base_image = BuildParam("flatpak_base_image", allow_none=True)
```

Comment:
I don't think you want to change build_request v1 here - that changes the behavior of Arrangement 5- systems.

Reply:
I think we're fine with making changes that only affect flatpak builds, as they are not working prior to arrangement 6 in any case.